remote/pom.xml

```diff
@@ -106,5 +122,10 @@
             <version>${dependency.jul.version}</version>
             <type>jar</type>
         </dependency>
+        <dependency>
```

Comment:
Is this really the right place to add these dependency? Since the query is performed on controller side anyway, we should be able to move the dependency to the `bco.dal.control` right?